tensorflow_model_optimization/g3doc/guide/clustering/clustering_example.ipynb

```diff
@@ -0,0 +1,515 @@
+{
```

Comment:
<p>From the cluster_wrapper.py code, it seems that clustering initialization occurs in the call to build(), which will happen during the the call to cluster_weights() for Sequential models. The same is true for Functional. </p><p> </p><p>So "cluster a whole model" is not applicable to the code below and is relevant to the above section instead. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Sorry, this version is pretty messy.</p>